rust/worker/src/tracing/opentelemetry_config.rs

```diff
@@ -49,4 +49,28 @@ pub(crate) fn init_otel_tracing(service_name: &String, otel_endpoint: &String) {
     tracing::subscriber::set_global_default(subscriber)
         .expect("Set global default subscriber failed");
     println!("Set global subscriber for {}", service_name);
+
+    // Add panics to tracing
+    let prev_hook = std::panic::take_hook();
+    std::panic::set_hook(Box::new(move |panic_info| {
```

Comment:
for my learning: does this panic hook get set system wide after this call?

Reply:
yes, it's a global hook for all threads (https://doc.rust-lang.org/std/panic/fn.set_hook.html)